### PR TITLE
Fix: Database remodelling

### PR DESCRIPTION
**Pull request by SundayMba**
## Description
this pull request introduces remodelling of database.

a new ER diagram was been generated due to the requirement we had, so i had to remodeled the database, more tables were added and unneccesary tables were removed.
migrations for all were generated.
​

## Related Issue (Link to issue ticket)
no link, since it is not an issue

​

## Motivation and Context

i needed to set up the database so that others can work using it as a base framework. all team members are depended on the tables.
​

## How Has This Been Tested?
yes

​

## Screenshots (if appropriate - Postman, etc):
i registered a user successfully, the information was stored in the database
​

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to change)
      ​

## Checklist:

- [x] My code follows the code style of this project.
- [] My change requires a change to the documentation.
- [] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [] I have added tests to cover my changes.
- [] All new and existing tests passed.

### CODE REVIEW

File: api/v1/models/base.py
Comment (bolexs) [inline api/v1/models/base.py:1]: Please do not comment out code, remove them if not needed

File: api/v1/models/permission.py
Comment (bolexs) [inline api/v1/models/permission.py:1]: Remove code that are not needed

**Review by bolexs — COMMENTED** [review]
Clean up the codebase

**Comment by SundayMba** [comment]
okay, on it now.

**Comment by SundayMba** [comment]
done boss.

**Comment by bolexs** [comment]
@Dev-wonderful will be merging this

**Pull request merged**